IfcCommunicationsAppliance Rail Import BIMQ 2021-07-21T07:21:13.7228368Z (#177)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/n/NumberOfTrafficInterfaces_09Z_XNzNLBnwwViB8ciE59/Documentation.md

> Indicates the types of traffic interfaces and their number in the device.

Reviewer: ```suggestion
Indicates the number of traffic interfaces on the device.
```
